README.md

```diff
@@ -35,15 +35,37 @@ HAMi is a sandbox and [landscape](https://landscape.cncf.io/?item=orchestration-
 HAMi provides device virtualization for several heterogeneous devices including GPU, by supporting device sharing and device resource isolation. For the list of devices supporting device virtualization, see [supported devices](#supported-devices)
 
 ### Device sharing
-
+HAMi supports:
 - Allows partial device allocation by specifying device memory.
-- Imposes a hard limit on streaming multiprocessors.
 - Permits partial device allocation by specifying device core usage.
 - Requires zero changes to existing programs.
 
 <img src="./imgs/example.png" width = "500" /> 
 
```

Comment:
I think it is necessary to focus on the meaning of the fields `nvidia.com/gpu`,`nvidia.com/gpumem`,`nvida.com/gpucores`. The detailed examples have special yaml and do not need to be presented here.